src/auth/auth-router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const manager = require('./auth-manager');
+const router = express.Router();
+
+/** hook up the routes **/
```

Comment:
From what I understand, JSDoc is only used on classes, fields, and methods (https://google.github.io/styleguide/jsguide.html#jsdoc).

Since the endpoint routers are technically just passing anonymous functions to ```router.METHOD()``` calls, they do not need JSDocs.